### ISSUE TITLE
McAfee LiveSafe false positives

### ISSUE THREAD

**Issue by git-nico** [portapps/brave-portable#22]
Hi Max,

love to be able to use Brave portable but I am running into false positives with McAfee LiveSafe.
When I use the installer I get the following
![virus](https://user-images.githubusercontent.com/51365811/58874302-b59c2080-86c8-11e9-9448-1e6a53f6019e.jpg)

And when I use the 7z archive I get:
![virus2](https://user-images.githubusercontent.com/51365811/58874345-dd8b8400-86c8-11e9-808e-8812c846a1aa.jpg)

I hope you can do something with this.

cheers
Nico

**Comment by crazy-max** [comment]
Hi @git-nico,

32bit or 64bit ?

**Comment by git-nico** [comment]
64bit  (brave-portable-win64-0.64.77-20)

**Comment by crazy-max** [comment]
Ok,

* Which McAfee product and version ?
* McAfee engine version ?

To help me, you can submit `brave-portable.exe` to McAfee [here](https://service.mcafee.com/webcenter/portal/oracle/webcenter/page/scopedMD/s55728c97_466d_4ddb_952d_05484ea932c6/Page29.jspx?articleId=TS102053). Go to Solution section and "I think I have a false positive".

**Comment by git-nico** [comment]
McAfee LiveSafe version 16.0, release name 16.0 R19
This Contains:
McAfee VirusScan version 22.4 (build 22.4.166), Engine version 3728.0

I will try to submit brave-portable.exe as well.

cheers,
Nico

**Comment by crazy-max** [comment]
Ok thanks @git-nico ! I have sent the wrapper as well. Please let me know.

**Comment by git-nico** [comment]
some extra info. McAfee does not detect anything when you scan the file brave-portable.exe or the complete folder. It only detects the false positive when you run brave-portable.exe.

**Comment by crazy-max** [comment]
Ok I have sent a hybrid analysis to check this behavior : https://www.hybrid-analysis.com/sample/6cb98e75af8c6946c172257d0fda6f5e0b28c5d386d3e9c5d2a8b7e030b4bd57

**Comment by git-nico** [comment]
Hi Max,

I think I owe you an apology. 
Because of the message(s) I got from the McAfee (screenshots) I thought it was caused by brave-portable.exe. I have done some more research and I can tell you, there is nothing wrong with the portable version. The false positives are caused by a rare combination of choices I made.
I am using the PortableApps.com Platform to launch all my portable apps. When I unzip the 7z archive into the portable apps folder in a new folder name "brave-portable" and I start brave using the PortableApps.com Platform, I get the McAfee warnings and brave-portable.exe is removed. 
The weird thing is that when I unzip the archive into a folder with a different name (I used the name brave-portable-win64-0.64.77-20) I do net get the McAfee warnings. 
I can even use the name "brave-portable" as the folder name when I first execute brave-portable.exe without the use of the PortableApps.com Platform. After that I can use the platform again without any warnings.
This has probably to do with the creation of brave-portable.sample.yml.

Anyway, I hope I have not wasted too much of your time.

Nico
